src/isort/mod.rs

```diff
@@ -469,9 +470,17 @@ fn sort_imports(block: ImportBlock) -> OrderedImportBlock {
     ordered
 }
 
+/// Confirms that the character after the first instance of the given part matches the match_char
+fn check_last_char(whole: &str, part: &str, match_char: &str) -> bool {
+    let idx = whole.find(part).unwrap();
+    let selected = whole.chars().nth(idx + part.len()).unwrap();
+    selected.to_string() == match_char
+}
```

Comment:
This will fail if there is a duplicate word before the desired word, I need to fix this before I am ready for a review.

Reply:
Sweet